site/docs/4.1/content/images.md

```diff
@@ -11,7 +11,7 @@ toc: true
 Images in Bootstrap are made responsive with `.img-fluid`. `max-width: 100%;` and `height: auto;` are applied to the image so that it scales with the parent element.
 
 <div class="bd-example">
-  <img data-src="holder.js/100px250" class="img-fluid" alt="Generic responsive image">
+  <img src="{% include holder.svg size="640x250" %}" class="img-fluid" alt="Generic responsive image">
```

Comment:
Can't we still keep the 100% functionality? Because it's quite handy not needing to specify the size in pixels.

Reply:
Wouldn't do that, in most cases images with dimensions are used. This makes it clear that the image is rescaled instead of being cropped.

Reply:
It seem's to work already with `{% include holder.svg size="100%x180" %}` I could change this, but I'm with Martijn, in the real world cases images with dimensions are used. What is the decision here?

Reply:
@XhmikosR, see my previous PR (example with `.mw-100`): https://deploy-preview-27633--twbs-bootstrap4.netlify.com/docs/4.1/utilities/sizing/

That's the result you 'll have when we allow `100%`, the image gets "cropped" from both sides if the viewport is resized and that is not the result we'll have with `jpeg, png,..` because we have a ratio with rasterised images.